rules/windows/defense_evasion_encoding_or_decoding_files_via_certutil.toml

```diff
@@ -12,32 +12,33 @@ control or exfiltration.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Encoding or Decoding Files via CertUtil"
 risk_score = 47
 rule_id = "fd70c98a-c410-42dc-a2e3-761c71848acf"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:certutil.exe and process.args:(-decode or -encode or /decode or /encode)
+process where event.type == "start" and
+  (process.name : "certutil.exe" or process.pe.original_file_name == "CertUtil.exe") and 
+  process.args : ("-decode", "-encode", "/decode", "/encode")
```

Comment:
```suggestion
  process.args : ("*decode", "*encode")
```

Reply:
not needed, but more resilient if theres something other than `-` and `/`

Reply:
ha you know what i'm going to say...
if we can avoid the leading `*` here, we should

Reply:
fine saint

Reply:
```suggestion
  process.args : ("?decode", "?encode")
```